### ISSUE TITLE
Setting description in invoice params

### ISSUE THREAD

**Issue by hieblmi** [fiatjaf/satdress#23]
Isn't the description also supposed to be set here?: https://github.com/fiatjaf/satdress/blob/5574a494edb36e88716107361522c8979e63ebac/makeinvoice.go#L67

**Comment by fiatjaf** [comment]
No, for Lightning Addresses use the `descriptionHash`, never `description`.
